LoRaEngine/modules/LoRaWanNetworkSrvModule/LoRaWan.NetworkServer/DefaultLoRaDataRequestHandler.cs

```diff
@@ -64,514 +65,541 @@ public class DefaultLoRaDataRequestHandler : ILoRaDataRequestHandler
 
         public async Task<LoRaDeviceRequestProcessResult> ProcessRequestAsync(LoRaRequest request, LoRaDevice loRaDevice)
         {
-            if (request is null) throw new ArgumentNullException(nameof(request));
-            if (loRaDevice is null) throw new ArgumentNullException(nameof(loRaDevice));
+            ArgumentNullException.ThrowIfNull(request, nameof(request));
+            ArgumentNullException.ThrowIfNull(loRaDevice, nameof(loRaDevice));
 
             List<Task> deferredTasks = null;
```

Comment:
I feel this method is a misnamed. It says the tasks are deferred, which is not true. They run concurrent with the main processing of the data request. I would call this `forkedTasks` and review related names.

Reply:
Renamed to `secondaryTasks` in 6b2d2025bb35c8620b01388c0439d3877d81acf4